tests/subtests/streamer-tests.js

```diff
@@ -57,43 +57,6 @@ var test = function(userToken) {
                 .then(() => { done(); })
                 .catch((err) => { done(err); });
         },
-        "testWithComponentSplitter": function(done) {
```

Comment:
Why is not the whole streamer-tests.js file removed?

Reply:
PR is updated with a commit to remove the file entirely, and to adapt oisp-tests.js accordingly.